web/lib/handlers/order.js

```diff
@@ -0,0 +1,33 @@
+const { asyncify } = require('async');
+const mastermind = require('../mastermind');
+const Order = require('../models/order');
+
+const get = params => // eslint-disable-line no-unused-vars
+  mastermind([
+    // eslint-disable-next-line no-unused-vars
+    asyncify(async (req, reply, cb) => {
+      const { userId } = req.user.id;
+      const { eventId } = req.params.eventId;
+      const query = Object.assign({}, req.query, { 'query[userId]': userId, 'query[eventId]': eventId });
+      const orders = await Order.get(query);
+      return reply(orders).code(200);
+    }),
+  ]);
+
+const post = params => // eslint-disable-line no-unused-vars
+  mastermind([
+    // eslint-disable-next-line no-unused-vars
+    asyncify(async (req, reply, cb) => {
+      const userId = req.user.user.id;
```

Comment:
Is it req.user.user.id or req.user.id ? https://github.com/CoderDojo/cp-zen-platform/pull/1252/files#diff-d35579e7e78f0621347d79ee5bfe4725R9

Reply:
sorry yeah its req.user.user.id. didnt change it for the get